Procfile

```diff
@@ -1 +1 @@
-web: PLOTLY_CONNECTOR_PORT=$PORT PLOTLY_CONNECTOR_STORAGE_PATH="/app/sample-storage" node ./dist/headless-bundle.js
+web: PLOTLY_CONNECTOR_PORT=$PORT PLOTLY_CONNECTOR_STORAGE_PATH="/app/sample-storage" PLOTLY_CONNECTOR_AUTH_ENABLED=false node ./dist/headless-bundle.js
```

Comment:
s/node/electron/

Reply:
@n-riesco Getting failure on deploys, when using `electron` instead of `node`:

```
2017-10-24T12:44:07.793994+00:00 app[web.1]: /app/node_modules/electron/dist/electron: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory
```

Keeping `node` until we figure this out..

Reply:
Oh I see, the `COPY` [here](https://github.com/plotly/falcon-sql-client/blob/b7a257ac615c7ab3a9c79aa47b63f17e2bf3be00/Dockerfile#L42) should be move back to its original position (i.e. the `COPY` must happen after `yarn run rebuild:modules:eklectron`).

Reply:
In that case, the `scripts/install-modules.js` file is not copied onto image, and `yarn run rebuild:modules:electron` fails. Error:

```
module.js:471 throw err; ^ Error: Cannot find module '/usr/src/app/scripts/install-modules.js' 
at Function.Module._resolveFilename (module.js:469:15) at Function.Module._load 
(module.js:417:25) at Module.runMain (module.js:604:10) at run (bootstrap_node.js:389:7) at 
startup (bootstrap_node.js:149:9) at bootstrap_node.js:504:3
```

Reply:
What doesn't `scripts/install-modules.js` get copied into the ocntainer?

Reply:
We need to do it manually, which we are doing here: https://github.com/plotly/falcon-sql-client/pull/208/files#diff-3254677a7917c6c01f55212f86c57fbfR42

Reply:
I'm now confused.  This is what I'd expect should work:

`PLOTLY_CONNECTOR_AUTH_ENABLED=false electron ./dist/headless-bundle.js`

and
 
```
COPY . /usr/src/app

# Install app dependencies
RUN yarn install
RUN yarn run rebuild:modules:electron
```

and  `PLOTLY_CONNECTOR_AUTH_ENABLED=false node ./dist/headless-bundle.js` should fail.

Reply:
Right - so Heroku doesn't use the `Dockerfile`, heroku just installs the files from `yarn.lock` and then runs the `heroku-postbuild` script: https://github.com/plotly/falcon-sql-client/blob/master/package.json#L11. There are other ways that you can customize this process here: https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process